resources/testharness.js

```diff
@@ -2105,8 +2105,6 @@ policies and contribution forms [3].
             this.set_status(status, message, stack);
             this.phase = this.phases.HAS_RESULT;
             this.done();
-        } finally {
-            this.current_test = null;
```

Comment:
Note that replacing this by:
```javascript
        tests.current_test = null
```
to match line 2078 isn't a good idea, because this would make assignment of assertion to the right test no more working correctly within `promise_test`, past the first micro-task.